breathe/renderer/sphinxrenderer.py

```diff
@@ -140,6 +145,61 @@ class PyClasslike(BaseObject, python.PyClasslike):
     pass
 
 
+# ----------------------------------------------------------------------------
+
+if cs is not None:
+    class CSharpObject(BaseObject, cs.CSharpObject):
+        pass
+
+    class CSharpVariable(BaseObject, cs.CSharpVariable):
+        pass
```

Comment:
Do we need to do this? If yes then it might also need to be done for the `php_classes` below? See below

Reply:
Yes, the ``BaseObject`` enables the scoped rendering. And yes, it should be done for the PHP domain at some point when it is updated for Sphinx >= 3.

Reply:
ok great